frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -277,20 +284,8 @@ export default memo(function Filters({ isLoading }: Props): JSX.Element {
 
         <Sort areFiltersDisabled={areFiltersDisabled} />
       </Wrapper>
-    </TooltipWrapper>
+    </Tooltip>
   );
-
-  function TooltipWrapper({ children }: { children: ReactElement }) {
-    if (areFiltersDisabled) {
-      return (
-        <Tooltip title="Please select an organism, tissue and at least one gene to use these filters.">
-          {children}
-        </Tooltip>
-      );
-    }
-
-    return <>{children}</>;
```

Comment:
switching between `<></>` and `<Tooltip></Tooltip>` causes the all subtree to unmount and remount to the DOM, thus reinstantiating all the react components, including all the useMemo calls, since all the components are now considered being mounted in the DOM for the first time